cg/meta/workflow/mutant.py

```diff
@@ -105,6 +105,27 @@ def get_sample_parameters(self, sample_obj: models.Sample) -> MutantSampleConfig
             ),
         )
 
+    def get_internal_NTCs(self, case_id: str) -> List[models.Sample]:
```

Comment:
```suggestion
    def get_internal_negative_controls(self, case_id: str) -> List[models.Sample]:
```

Reply:
updated according to suggestion and changed in the code where function is used